files/image_config/monit/memory_checker

```diff
@@ -59,6 +69,81 @@ def get_command_result(command):
 
     return command_stdout.strip()
 
+def get_container_id(container_name):
+    """Gets full container ID of the specified container
+    Args:
+        container_name: A string indicates the name of specified container.
+    Returns:
+        container_id: A string indicates the full ID of specified container.
+    """
+    container_id = ""
+
+    get_container_info_cmd = "docker ps --no-trunc --filter name={}".format(container_name)
```

Comment:
>container_name [](http://example.com/codeflow?start=76&length=14)

This is a obvious vulnerability for command line injection. Malicious user could easily give a bad container name with special char. Please valid input, and never concat a command lines. Keep tokens in an array. 
@maipbui to help review.

Reply:
I agree with Qi's comment. Need command to be a list of strings, instead of strings
e.g.
```
    command = ["docker", "stats", "--no-stream", "--format", "{{.MemUsage}}", container_name]
    command_stdout = get_command_result(command)
```

Reply:
Fixed